### DISCUSSION TITLE
Checking if Coloring file exists...

### DISCUSSION THREAD

**Discussion by hosseinnegahban** [mdolab/dafoam#262] [Bug]
Hey guys,

I am running a 3D wing case in DAFoam but every time I run the case, no matter parallel with 4 core or single core, the solution converges to the target Cl and finds the corresponding alpha, but when coloring method starts, the solution stops in "Checking if coloring file exists..." and there is no progress at all. Could you please help me  with this question?

I have also modified mesh, but it is the same...
I have also attached the log file for your consideration.
[logOpt.txt](https://github.com/mdolab/dafoam/files/8066346/logOpt.txt)

Thank you in advance,
Hossein

**Comment by friedenhe** [comment]
Check this: https://dafoam.github.io/mydoc_get_started_faq.html#how-many-cpu-cores-to-use-and-how-much-memory-does-it-need
